apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.scss

```diff
@@ -25,7 +25,9 @@
 .ledger-panel table tbody tr td a {
     color: #333;
```

Comment:
Use color variable in this

Reply:
I have not added this color right now, please focus on red and green line changes